packages/model-fragments/tests/integration/app_test.js

```diff
@@ -3,7 +3,7 @@ var app;
 QUnit.module("integration - Application");
 
 test("the model fragments initializer causes no deprecations", function() {
-  expectNoDeprecation();
+  ok(true); // expectNoDeprecation();
```

Comment:
Yeap :). Now that you've brought the `getOwner` polyfill in, I would expect that if you undid that change for the tests, then reapplied your container/registry => getOwner changes, it should work. I can't see the commit anymore, but are two usages of registry ([both usages](https://github.com/lytics/ember-data-model-fragments/blob/557f4a288a6966fb4e8df48a80871ae11a17eea0/packages/model-fragments/lib/fragments/ext.js#L319-L327) and two usages of container ([1](https://github.com/lytics/ember-data-model-fragments/blob/557f4a288a6966fb4e8df48a80871ae11a17eea0/packages/model-fragments/lib/fragments/ext.js#L320-L329) and [2](https://github.com/lytics/ember-data-model-fragments/blob/master/packages/model-fragments/lib/fragments/transforms/array.js#L64)).